testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/core/basic/DuplicateDeploymentTest.java

```diff
@@ -27,6 +28,7 @@
  */
 @RunWith(Arquillian.class)
 @RunAsClient
+@Category(NotForBootableJar.class)
```

Comment:
Some description with reason would be useful (eg add `// no log check support for bootable-jar in RESTEasy TS so far`)